_sass/description.scss

```diff
@@ -0,0 +1,64 @@
+body {
+    // background-image: linear-gradient(#3F51B5, #1A237E);
```

Comment:
If you don't need this I'd remove it. It's usually not a good idea to push commented code.